src/intl/en/page-languages.json

```diff
@@ -11,6 +11,7 @@
   "page-languages-want-more-header": "Want to see ethereum.org in a different language?",
   "page-languages-want-more-link": "Translation Program",
   "page-languages-want-more-paragraph": "ethereum.org translators are always translating pages in as many languages as possible. To see what they're working on right now or to sign up to join them, read about our",
+  "page-languages-want-more-content": "ethereum.org translators are always translating pages in as many languages as possible. To see what they're working on right now or to sign up to join them, read about our <a href=\"/contributing/translation-program\">Translation Program</a>.",
```

Comment:
Oh good catch... we _should_ convert strings with links to use the `htmr` format, so the label for the link can be translated in the appropriate context (and more importantly it can be moved within the sentence, if appropriate for that language)... I added this as an update, but then changed my mind to limit how many strings we break.

If we're going to make a push to get all of this translated before launch, I might suggest we add this `-content` string back and get rid of the `-link` and `-paragraph` strings that are currently in production.

Reply:
Actually, this particular item needs a more complex component than just an anchor link... will keep this as-in for now and we can discuss separately how we'd like to approach these going forward.